casper/src/test/scala/coop/rchain/casper/merging/MergeNumberChannelSpec.scala

```diff
@@ -339,8 +347,8 @@ class MergeNumberChannelSpec extends AnyFlatSpec {
         DeployTestInfo(rhoChange(10), 10L, "0x21"), // +10
         DeployTestInfo(rhoChange(-20), 10L, "0x22") // -20
       ),
-      expectedRejected = Set.empty,
-      expectedFinalResult = 10
+      expectedRejected = Set(makeSig("0x11")),
+      expectedFinalResult = 15
```

Comment:
This fix is because conflict resolution item now is not a branch but a single deploy chain index.

Reply:
This doesn't make sense. Left branch contains +5 change. With what is 0x11 conflicting?